src/shared/invite-response/invite-response.tsx

```diff
@@ -226,6 +244,285 @@ const InviteResponse: FC<InviteResponse> = ({
 								proposeNewTime={proposeNewTime}
 							/>
 					  )}
+
+				{invite[0]?.comp[0].loc && (
+					<Row width="fill" mainAlignment="flex-start" padding={{ top: 'large' }}>
+						<Tooltip placement="left" label={t('tooltip.location', 'Location')}>
+							<Row mainAlignment="flex-start" padding={{ right: 'small' }}>
+								<Icon size="large" icon="PinOutline" />
+							</Row>
+						</Tooltip>
+
+						<Row takeAvailableSpace mainAlignment="flex-start">
+							<Tooltip placement="right" label={invite[0]?.comp[0]?.or.a}>
+								<Text size="medium" overflow="break-word" style={{ fontSize: '16px' }}>
+									{invite[0]?.comp[0].loc}
+								</Text>
+							</Tooltip>
+						</Row>
+					</Row>
+				)}
+
+				{chatLink && (
+					<Row width="fill" mainAlignment="flex-start" padding={{ top: 'large' }}>
+						<Tooltip placement="left" label={t('tooltip.chat', 'Virtual Chat')}>
+							<Row mainAlignment="flex-start" padding={{ right: 'small' }}>
+								<Icon size="large" icon="VideoOutline" />
+							</Row>
+						</Tooltip>
+
+						<Row takeAvailableSpace mainAlignment="flex-start">
+							<Tooltip placement="right" label="link">
+								<LinkText
+									color="primary"
+									size="medium"
+									onClick={(): void => setMaxReqParticipantsToShow(requiredParticipants.length)}
+									overflow="break-word"
+								>
+									Chat&#39;s room
+								</LinkText>
+							</Tooltip>
+						</Row>
+					</Row>
+				)}
+
+				{haveEquipment && (
+					<Row width="fill" mainAlignment="flex-start" padding={{ top: 'large', bottom: 'small' }}>
+						<Tooltip placement="left" label={t('tooltip.equipment', 'Equipment')}>
+							<Row mainAlignment="flex-start" padding={{ right: 'small' }}>
+								<Icon size="large" icon="BriefcaseOutline" />
+							</Row>
+						</Tooltip>
+						<Row takeAvailableSpace mainAlignment="flex-start">
+							<Tooltip placement="right" label="Equipment name">
```

Comment:
label should be translated

Reply:
The section about the equipment is static, for now i've just prepared the structure (have to dynamize label, text and condition when equipment management is gonna be implemented)